site/components/payment-streams/EditStream.js

```diff
@@ -259,27 +266,31 @@ const EditFundingAddress = function ({ stream }) {
 }
 
 const EditStream = function () {
-  const router = useRouter()
+  const {
+    query: { id: streamId }
+  } = useRouter()
+  const router = useIntlRouter()
   const { active } = useWeb3React()
-  const { streamId } = router.query
-  const { t } = useTranslation('payment-streams')
+  const t = useTranslations('payment-streams-util')
   const { streams = { outgoing: [] }, isLoading } = useStreams()
 
-  if (!active) {
-    router.push('/payment-streams')
-    return null
-  }
+  const stream = useMemo(
+    () => streams.outgoing.find(s => s.id === streamId),
+    [streamId, streams.outgoing]
+  )
+  useEffect(
+    function () {
```

Comment:
Let's name this function too.

Reply:
here: https://github.com/hemilabs/pure.finance/pull/1/commits/6c05f4bd01318c255d6de41f88ebe95d30fa1892